src/ios/JXcoreExtension.m

```diff
@@ -28,13 +28,118 @@
 #import "JXcoreExtension.h"
 #import "THEAppContext.h"
 
+#import "JXcore.h"
+
 // JXcoreExtension implementation.
 @implementation JXcoreExtension
 
 // Defines methods.
 - (void)defineMethods
 {
-    [[THEAppContext singleton] defineJavaScriptExtensions];
+  THEAppContext *theApp = [THEAppContext singleton];
+
+  // Export the public API to node
+
+  // StartBroadcasting
+  [JXcore addNativeBlock:^(NSArray * params, NSString * callbackId) 
+  {
+    NSLog(@"jxcore: startBroadcasting");
```

Comment:
As explained in frightening detail [here](http://doing-it-wrong.mikeweller.com/2012/07/youre-doing-it-wrong-1-nslogdebug-ios.html) we really aren't supposed to use NSLog for anything but earth shattering errors. Shouldn't we be using ASL or something equivalent? I'm sure there are slightly more sophisticated frameworks that define the necessary APIs in a friendlier way than ASL and then dump into ASL. So suggest the framework of your choice but I'm pretty sure NSLog isn't our friend here.

I understand if you don't want to fix this right now but then please open an issue and assign it to yourself.